gammapy/catalog/hawc.py

```diff
@@ -196,7 +169,114 @@ class SourceCatalog2HWC(SourceCatalog):
     description = "2HWC catalog from the HAWC observatory"
     """Catalog description"""
 
-    source_object_class = SourceCatalogObject2HWC
+    source_object_class = SourceCatalogObjectHWC
```

Comment:
Here you should keep `SourceCatalogObject2HWC` as this is the object used by this class.